src/stateful_check/core.clj

```diff
@@ -131,7 +132,22 @@
                (update try-result :result-data
                        merge {:commands commands, :options options, :specification spec})))))))))
 
-(defn- print-sequence [commands stacktrace? messages]
+(defn- print-failures [handle failures]
+  (doseq [{:keys [message events]} (get failures handle)]
+    (if-not (seq events)
+      (println "   " message)
+      (doseq [{:keys [message] :as event} events]
+        (when message
+          (println "     " message))
+        (doseq [detail [:expected :actual]]
+          (when (contains? event detail)
```

Comment:
I think I got mislead by this comment here: https://github.com/r0man/cider-nrepl/blob/stateful-check/src/cider/nrepl/middleware/test.clj#L114

But I think, since errors thrown outside of an assertion (faults) are catched by the quick-check function we don't need it. I think it was wrong anyway. The key seems to be present for faults as well, it's value is just nil. I removed it.